code/modules/projectiles/guns/manufacturer/frontier_import/ballistics.dm

```diff
@@ -0,0 +1,229 @@
+/obj/item/gun/ballistic/automatic/pistol/mauler
+	name = "Mauler machine pistol"
+	desc = "A full auto machine pistol originating from the Shoal. Insane stopping power, but mostly useless outside of CQC with unarmored targets. Chambered in 9mm."
```

Comment:
"An automatic machine pistol originating from the Shoal. Impressive volume of fire with abysmal accuracy, lackluster armor penetration, and limited magazine size render it mostly useless outside of very close quarters. Chambered in 9mm."